test/scale/label-nodes.sh

```diff
@@ -17,11 +18,17 @@ fi
 
 for node in $(kubectl get nodes -o name);
 do
+    node_count=$((node_count + 1))
+    echo $node_count
     echo "Current : $node"
     node_name="${node##*/}"
     echo "Apply label to the node"
     kubectl label node $node_name connectivity-test=true
     kubectl label node $node_name scale-test=true
+    if [ $node_count -lt 3 ]; then
+        kubectl label node $node_name netperf=true
```

Comment:
Since we are using the same cluster for scale-test, are we removing these labels ? This is a one time thing ?

Reply:
The nodes that get removed after scaling will need to be relabeled after the 1000 are reprovisioned